lib/migrate_with_cleared_schema_cache.rb

```diff
@@ -0,0 +1,20 @@
+module MigrateWithClearedSchemaCache
```

Comment:
I feel like this should be namespaced to the gem namespace.

Reply:
Done